### DISCUSSION TITLE
Nova Impersonation

### DISCUSSION THREAD

**Discussion by nikoo-eshratabadi** [laravel/nova-issues#6415] [Q&A] [answered]
- Laravel Version: 10.10
- Nova Version: 4.25.1
- PHP Version: 8.1.23

I'm having trouble using the built-in nova impersonation feature. Unfortunately, it's not visible inline and only shows up in the drop down menu. Is it possible to show it in the table rows as an icon?

I simply added the trait to my user model and tested.

**Comment by crynobone** [comment]
Such feature is not offered by Laravel Nova by default

**Reply by nikoo-eshratabadi** [reply]
ow ok thanks for the reply

**Reply by nikoo-eshratabadi** [reply]
is there any way i can start the impersonation myself? can i define a custom action and start if there?

**Reply by nikoo-eshratabadi** [reply, marked as answer]
i actually were able to do this . i just sent a http post request to a route i defined with csrf token
